vignettes/rhino-fluent.Rmd

```diff
@@ -0,0 +1,321 @@
+
+## Create an initial `rhino` application
+
+Now that you have the package installed, let's use `rhino` to create an initial application for us which we can use for the next steps of our project.
+
+Creating a new Rhino application can be done in two ways - by using the **Create Project** functionality on RStudio IDE or by running **`rhino::init()`** function from the console.
+We'll discuss more about this below 👇
+
+### 👉 Create an application using the RStudio wizard
+
+If you use RStudio, probably the easiest way to create a new Rhino application is to simply use Create `New Project` feature.
+Once Rhino is installed, it will be automatically added as one of the options in RStudio:
+
+Choose it, input the new project name and you are ready to go 👍
+
+![](https://appsilon.github.io/rhino/articles/tutorial/images/rstudio_wizard.png)
+
+### 👉 Create an application using `rhino::init()`
+
+Creating a Rhino application is possible in the R console by running the `init` function:
+
+``` r
+rhino::init("RhinoApplication")
+```
+
+There are two things you need to know when choosing this way of initializing your application:
+
+-   **Rhino will not change your working directory** - You need to either open a new R session in your new application directory or manually change the working directory.
+
+```         
+setwd("./RhinoApplication")
+```
+
+-   **Rhino relies on options added to the projects `.Rprofile` file -** The most robust way to make sure it was correctly sourced is to simply restart the R session.
+
+A result of both paths will be an initial Rhino application with the following structure:
+
+```         
+.
+├── RhinoApplication.Rproj
+├── app
+│ ├── js
+│ │ └── index.js
+│ ├── logic
+│ │ └── __init__.R
+│ ├── main.R
+│ ├── static
+│ │ └── favicon.ico
+│ ├── styles
+│ │ └── main.scss
+│ └── view
+│     └── __init__.R
+├── app.R
+├── config.yml
+├── dependencies.R
+├── renv.lock
+├── rhino.yml
+└── tests
+    ├── cypress
+    │ └── integration
+    │     └── app.spec.js
+    ├── cypress.json
+    └── testthat
+        └── test-main.R
+```
+
+If you want to know more about it, check [this](https://appsilon.github.io/rhino/articles/explanation/application-structure.html) document.
+
+### 🚀 Running the initial application
+
+Now, once you are all set up, let's run it:
+
+```         
+shiny::runApp()
+```
+
+And here is what you should be seeing right now 👇
+
+![](https://appsilon.github.io/rhino/articles/tutorial/images/init_application.png){width="620" height="268"}
+
+## Let's install `shiny.fluent` for our project
+
+💡The rhino project you just started uses `renv` for dependency management.
+You can now install any other package in the same project and include it in the `renv.lock` file.
+
+Here's how you can install the `shiny.fluent` package 👇
+
+``` r
+install.packages("shiny.fluent")
+```
+
+Now that the package is installed, let us use it for building our very first `rhino` module.
```

Comment:
This section is incorrect.
1. Package should be installed using `renv::install` (it has benefits like adding it to the cache)
2. It misses the whole part of adding the package to `dependencies.R` and updating `renv.lock`.